### DISCUSSION TITLE
What's the difference?

### DISCUSSION THREAD

**Discussion by marcient** [vercel/community#689] [Show & Tell]
Hello,

I'm look into a headless wordpress set up and came across these 3 options:

1. https://vercel.com/support/articles/wordpress-with-vercel
2. https://www.strattic.com
3. https://getshifter.io

Strattic and Shifter seem like a more focused solution. 

Any opinions why using vercel would be the way to go from a tech or feature perspective (I'm not factoring in price in my decision)?

Thank you in advance for your insights!

**Comment by amyegan** [comment]
Deciding which of these options is best for you depends on your project, the features you need, and what you are comfortable using. I would recommend looking at the features of Strattic and Shifter and compare them with other WordPress hosting options so you can decide what would work best for you.
